app/templates/jinja2/email/signup.html

```diff
@@ -19,12 +19,11 @@
 <p>Thank you for signing up for Humans vs Zombies {{ game }}!</p>
 <p><a href="{{ site_url }}{{ url('signup', args=[signup_invite.id]) }}">Please finish signing up to our website here.</a></p>
 <p>Make sure to read and agree to the waiver, as well as read the rules of the game.</p>
-<p>The game will begin on <strong>Monday, November 5th @ 12:00 AM</strong>. If you have any friends who have never played before and would like to join, please send them to one of our signup booths before Friday, November 2nd @ 5 PM. Alternatively, they can contact us at <a href="mailto:uwhumansvszombies@gmail.com">uwhumansvszombies@gmail.com</a> via email.</p>
-<p>At <strong>5:00 PM on Sunday, November 4th</strong> in STC on the UWaterloo Campus, we will be hosting a practice mission to practice for the weeklong game. It will help introduce rules, have a little bit of fun and get the hang of the game. Please bring a bandana and any socks or blasters to use.</p>
-<p>At <strong>11:00 PM on Sunday, November 4th</strong>, you will receive an email whether you are a HUMAN or a ZOMBIE. If you would like to be a zombie, please check the box when you are signing up for the game.</p>
+<p>The game will begin on <strong>Monday, March 11th @ 12:00 AM</strong>. If you have any friends who have never played before and would like to join, please send them to one of our signup booths before Friday, March 8th @ 5 PM. Alternatively, they can contact us at <a href="mailto:uwhumansvszombies@gmail.com">uwhumansvszombies@gmail.com</a> via email.</p>
```

Comment:
We should really add a better way to let people edit templates without committing them to version control, as these dates/times are going to change every term and committing these changes doesn't add anything to the codebase. Would you mind leaving this out of the PR for now? Will file issue like the above situation with the countdown.